cocos/platform/openharmony/modules/CCCanvasRenderingContext2D-openharmony.cpp

```diff
@@ -245,13 +227,25 @@ class CanvasRenderingContext2DImpl {
                     bool /* smallCaps */) {
         _fontName = fontName;
         _fontSize = static_cast<int>(fontSize);
-        std::string fontPath;
-        if (!_fontName.empty()) {
-            const char* fontFamilies[1];
-            fontFamilies[0] = fontName.c_str();
-            OH_Drawing_SetTextStyleFontFamilies(_textStyle, 1, fontFamilies);
-            OH_Drawing_SetTextStyleLocale(_textStyle, "en");
+        
+        const auto &fontInfoMap = getFontFamilyCollectionMap();
+        auto it = fontInfoMap.find(_fontName);
+        if (it != fontInfoMap.end()) {
+            _fontCollection = it->second;
+        } else {
+            _fontCollection = OH_Drawing_CreateSharedFontCollection();
```

Comment:
If it run to this condition, what is the font displayed by that fontCollection?

Reply:
will use system font

Reply:
Got it.